### PR TITLE
DOCSUP-1394

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

### CODE REVIEW

File: docs/en/operations/system-tables.md [hunk 1]
```diff
@@ -1339,4 +1339,43 @@ Columns:
 
 If the storage policy contains more then one volume, then information for each volume is stored in the individual row of the table.
 
+##system.roles {#system_tables-roles}
+Contains information about [role](../operations/access-rights.md#role-management) that can be granted to a user account.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
Contains information about configured [roles](../operations/access-rights.md#role-management).
```

File: docs/en/operations/system-tables.md [hunk 2]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+Columns:
+- name ([String](../sql-reference/data-types/string.md)) — Role name.
+- id ([UUID](../sql-reference/data-types/uuid.md)) — Role ID.
+- storage ([String](../sql-reference/data-types/string.md)) — Storage of roles. The folder path on disk is set in the config file — `access_control_path` parameter.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
- storage ([String](../sql-reference/data-types/string.md)) — Path to the storage of roles. Configured in the `access_control_path` parameter.
```

File: docs/en/operations/system-tables.md [hunk 3]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+##system.role_grants {#system_tables-role_grants}
+Contains the role grants for users and roles. For adds entries to this table use 'GRANT role TO user'.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
Contains the role grants for users and roles. To add entries to this table, use `GRANT role TO user`.
```

File: docs/en/operations/system-tables.md [hunk 4]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+Columns:
+- user_name ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — User name.
+- role_name ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — Role name.
+- granted_role_name ([String](../sql-reference/data-types/string.md)) — Role name when it is granted to another role. In order for a role granted a different role, use 'GRANT role1 TO role2'.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
- granted_role_name ([String](../sql-reference/data-types/string.md)) — Name of role granted to the `role_name` role. To grant one role to another one use `GRANT role1 TO role2`.
```

Comment (BayoNet) [reply docs/en/operations/system-tables.md]: Does the `granted_role_name` is about a role granted to another role only? I suppose that we talk about the role granted to a user account or a role.

File: docs/en/operations/system-tables.md [hunk 5]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+- granted_role_is_default ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Granted role by default. Possible values:
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
- granted_role_is_default ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `granted_role` is a default role. Possible values:
```

File: docs/en/operations/system-tables.md [hunk 6]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+    -   1 — The role has default privileges.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
    -   1 — `granted_role` is a default role.
```

File: docs/en/operations/system-tables.md [hunk 7]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+    -   0 — The role without default privileges.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
    -   0 — `granted_role` is not a default role.
```

File: docs/en/operations/system-tables.md [hunk 8]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+- granted_role_name ([String](../sql-reference/data-types/string.md)) — Role name when it is granted to another role. In order for a role granted a different role, use 'GRANT role1 TO role2'.
+- granted_role_is_default ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Granted role by default. Possible values:
+    -   1 — The role has default privileges.
+    -   0 — The role without default privileges. 
+- with_admin_option ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role with administrator options. Possible values:
+    -   1 — The role has administrator options.
+    -   0 — The role without administrator options. 
+
+##system.current_roles {#system_tables-current_roles}
+Contains the current roles of the current user, it is a different table for different users (like system.settings), 'SET ROLE' changes the contents of this table.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
Contains active roles of a current user. `SET ROLE` changes the contents of this table.
```

File: docs/en/operations/system-tables.md [hunk 9]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+Columns:
+ - role_name ([String](../sql-reference/data-types/string.md))) — Role name.
+ - with_admin_option ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role with administrator options.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: Isn't it a flag that shows whether the role has `ADMIN OPTION`?

File: docs/en/operations/system-tables.md [hunk 10]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+ - is_default ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role by default.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: This is the flag too.

File: docs/en/operations/system-tables.md [hunk 11]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+##system.enabled_roles {#system_tables-enabled_roles}
+Contains: a) current roles of the current user + b) granted roles for a) + granted-roles for b)+ ..., i.e. in fact these are all active roles at the moment.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: Didn't understand at all.

File: docs/en/operations/system-tables.md [hunk 12]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+
+Columns:
+- role_name ([String](../sql-reference/data-types/string.md))) — Role name.
+- with_admin_option ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role with administrator options.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: Flag?

File: docs/en/operations/system-tables.md [hunk 13]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+- is_current ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role of current user.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: Flag?

File: docs/en/operations/system-tables.md [hunk 14]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+- is_default ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Role by default.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]: Flag?

File: docs/en/operations/system-tables.md [hunk 15]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+##system.roles {#system_tables-roles}
+Contains information about configured [roles](../operations/access-rights.md#role-management).
+
+Columns:
+- name ([String](../sql-reference/data-types/string.md)) — Role name.
+- id ([UUID](../sql-reference/data-types/uuid.md)) — Role ID.
+- storage ([String](../sql-reference/data-types/string.md)) — Path to the storage of roles. Configured in the `access_control_path` parameter.
+
+##system.role_grants {#system_tables-role_grants}
+Contains the role grants for users and roles. To add entries to this table, use 'GRANT role TO user'.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
Contains the role grants for users and roles. To add entries to this table, use `GRANT role TO user`.
```

File: docs/en/operations/system-tables.md (hunk 14 above)
Comment (adevyatova) [reply docs/en/operations/system-tables.md]: Fixed

File: docs/en/operations/system-tables.md (hunk 11 above)
Comment (adevyatova) [reply docs/en/operations/system-tables.md]: Fixed

File: docs/ru/operations/system-tables.md [hunk 16]
```diff
@@ -1249,4 +1249,43 @@ Cодержит информацию о дисках, заданных в [ко
 
 Если политика хранения содержит несколько томов, то каждому тому соответствует отдельная запись в таблице.
 
+##system.roles {#system_tables-roles}
+Содержит сведения о настроенных [ролях](../operations/access-rights.md#role-management).
```
Comment (BayoNet) [inline docs/ru/operations/system-tables.md]:
```suggestion
Содержит сведения о [ролях](../operations/access-rights.md#role-management).
```

File: docs/ru/operations/system-tables.md [hunk 17]
```diff
@@ -1249,4 +1249,43 @@ Cодержит информацию о дисках, заданных в [ко
...
+- `id` ([UUID](../sql-reference/data-types/uuid.md)) — ID роли.
+- `storage` ([String](../sql-reference/data-types/string.md)) — Путь к хранилищу ролей. Настраивается в параметре `access_control_path`.
+
+##system.role_grants {#system_tables-role_grants}
+Содержит [гранты](../sql-reference/statements/grant.md) ролей для пользователей и ролей. Чтобы добавить записи в эту таблицу, используйте команду `GRANT role TO user`.
+
+Столбцы:
+- `user_name` ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — Имя пользователя.
+- `role_name` ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — Имя роли.
+- `granted_role_name` ([String](../sql-reference/data-types/string.md)) — Имя роли, предоставленной роли `role_name`. Чтобы предоставить одну роль другой используйте `GRANT role1 TO role2`.
```
Comment (BayoNet) [inline docs/ru/operations/system-tables.md]:
```suggestion
- `granted_role_name` ([String](../sql-reference/data-types/string.md)) — Имя роли, назначенной для роли `role_name`. Чтобы назначить одну роль другой используйте `GRANT role1 TO role2`.
```

File: docs/ru/operations/system-tables.md [hunk 18]
```diff
@@ -1249,4 +1249,43 @@ Cодержит информацию о дисках, заданных в [ко
...
+##system.current_roles {#system_tables-current_roles}
+Содержит активные роли текущего пользователя. `SET ROLE` изменяет содержимое этой таблицы.
+
+Столбцы:
+ - `role_name` ([String](../sql-reference/data-types/string.md))) — Имя роли.
+ - `with_admin_option` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Флаг, который показывает, обладает ли `current_role` роль привилегией `ADMIN OPTION`.
+ - `is_default` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) —  Флаг, который показывает, является ли `current_role` ролью по умолчанию. 
+
+##system.enabled_roles {#system_tables-enabled_roles}
+Содержит все активные роли на данный момент, включая текущую роль текущего пользователя и предоставленные роли для текущей роли.
```
Comment (BayoNet) [inline docs/ru/operations/system-tables.md]:
```suggestion
Содержит все активные роли на данный момент, включая текущую роль текущего пользователя и  роли, назначенные для текущей роли.
```

File: docs/en/operations/system-tables.md [hunk 19]
```diff
@@ -1339,4 +1339,43 @@ Columns:
...
+ - `with_admin_option` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `current_role` is a role with `ADMIN OPTION` privilege.
+ - `is_default` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `current_role` is a default role.
+
+##system.enabled_roles {#system_tables-enabled_roles}
+Contains all active roles at the moment, including current role of the current user and granted roles for current role. 
+
+Columns:
+- `role_name` ([String](../sql-reference/data-types/string.md))) — Role name.
+- `with_admin_option` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `enabled_role` is a role with `ADMIN OPTION` privilege.
+- `is_current` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `enabled_role` is a role of current user.
```
Comment (BayoNet) [inline docs/en/operations/system-tables.md]:
```suggestion
- `is_current` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Flag that shows whether `enabled_role` is a current role of a current user.
```

File: docs/ru/operations/system-tables.md [hunk 20]
```diff
@@ -1249,4 +1249,43 @@ Cодержит информацию о дисках, заданных в [ко
...
+
+Столбцы:
+- `role_name` ([String](../sql-reference/data-types/string.md))) — Имя роли.
+- `with_admin_option` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Флаг, который показывает, обладает ли `enabled_role` роль привилегией `ADMIN OPTION`. 
+- `is_current` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Флаг, который показывает, является ли `enabled_role` ролью текущего пользователя.
```
Comment (BayoNet) [inline docs/ru/operations/system-tables.md]:
```suggestion
- `is_current` ([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges)) — Флаг, который показывает, является ли `enabled_role` текущей ролью текущего пользователя. 
```

**Pull request merged**